draft-ietf-opsawg-rfc5706bis.md

```diff
@@ -969,9 +969,9 @@ DM         DM        DM     --> concrete/detailed model
    Should there be aggregated status indicators based on concatenation
    of propagated Faults from a given domain or device?
 
-   SNMP notifications and syslog messages can alert an operator when an
+   Notifications and alert messages (e.g., SNMP, syslog, or New Protocol specific mechanism) can alert an operator when an
```

Comment:
You already have the word "alert" there.  What about just using "Notifications (e.g., ...)"?

Reply:
Works for me. Fixed in [Notifications/Alerts](https://github.com/IETF-OPSAWG-WG/draft-opsarea-rfc5706bis/pull/174/commits/41510cc35c68891947e3d46e7199b28c14a03f3c)